pkg/local_object_storage/metabase/version.go

```diff
@@ -71,3 +80,26 @@ func getVersion(tx *bbolt.Tx) uint64 {
 
 	return 0
 }
+
+var migrateFrom = map[uint64]func(*DB, *bbolt.Tx) error{
+	2: migrateFrom2Version,
```

Comment:
won't it be easier to just make migration steps one-by-one? iiuc following version funcs will have to call previous ones, right?

Reply:
It should one-by-one, but we may strip older handlers (like we don't have handlers for 0 and 1 already). We don't have a lot of versions, so 10 `nil` pointers are not that big of a problem, but a map works fine as well (check N, check N+1, etc).

Reply:
ok will see in practice

Reply:
there is a single migration for now, not sure how it will look further, can be changed in the future